src/pcm/SoxrResampler.cxx

```diff
@@ -39,9 +39,16 @@ static constexpr unsigned long SOXR_DEFAULT_RECIPE = SOXR_HQ;
  */
 static constexpr unsigned long SOXR_INVALID_RECIPE = -1;
 
+/**
+ * Special value for custom recipe.
+ */
+static constexpr unsigned long SOXR_CUSTOM_RECIPE = -2;
+
+static soxr_io_spec_t soxr_io;
```

Comment:
I don't like the variable name; this is for storing the custom settings and is unused otherwise, but the name doesn't tell that.

Reply:
Is soxr_io_custom_recipe better ?